src/quantile.rs

```diff
@@ -182,6 +182,17 @@ where
     S: Data<Elem = A>,
     D: Dimension,
 {
+    /// Finds the first index of the minimum value of the array.
+    ///
+    /// Returns `None` if any of the pairwise orderings tested by the function
+    /// are undefined. (For example, this occurs if there are any
+    /// floating-point NaN values in the array.)
+    ///
+    /// Returns `None` if the array is empty.
```

Comment:
Can we add a doc test example to both argmin/argmax? `D::Pattern` is the right signature, but it might feel a bit obscure to a new comer looking at the docs for the first time (even more if they don't know `ndarray` and they have to dig) - a simple example with a 2d array showing that it's just a `(usize, usize)` would go a long way to make it more obvious :pray: